src/main/java/org/generationcp/middleware/domain/etl/MeasurementData.java

```diff
@@ -35,6 +35,8 @@
 	private String dataType;
 	private Integer phenotypeId;
 	private MeasurementVariable measurementVariable;
+	private Integer instanceDataId;
```

Comment:
As discussed, let's try to make "phenotypeId" generic in order to remove the need for dupe variable fields like `instanceDataId`

Reply:
Done